### PR TITLE
First Design / Branding Pass

### PR DESCRIPTION
**Pull request by brianfeister**
This PR introduces our first pass of "real design changes" and supersedes #29 

- [x] Header / flyout mobile nav (with "close" button ✨ )
- [x] Introduce bg illustration to complement existing branding (warm / vibrant), carefully CSS all aspect ratios for optimum image placement
- [x] Add our only "real" nav items `Add an Event` and `About` (Note: that `Add an Event` should lead to `/register` for non-authenticated users, for authenticated users, it will take them to the `/admin/add-event` or something
- [x] Fixes responsive brokenness via `<meta name="viewport" content="width=device-width, initial-scale=1"/>`
- [x] Fix "dark mode" table and use standard daisy ui for table

Closes https://github.com/meetnearme/api/pull/29/files

GIF of the responsive behaviors (the z-index issue seen in this GIF is fixed at https://github.com/meetnearme/api/pull/32/commits/04a7a2d0e47b828bb98f62b5774364c839dc73d4)

Screen_Recording_2024-03-26_at_11 25 41 AM

### CODE REVIEW

File: static/assets/logo.svg [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<svg width="223" height="259" viewBox="0 0 223 259" fill="none" xmlns="http://www.w3.org/2000/svg">
+<g clip-path="url(#clip0_1_2)">
+<path style="mix-blend-mode:multiply" d="M154.305 152H37C40.6554 135.649 54.2801 138.446 65.2463 131.978C86.1818 119.632 82.8587 107.619 82.8587 102.613C82.8587 97.6077 79.2033 94.6045 77.2095 92.6023C75.2156 90.6001 70.5633 87.5969 65.2463 76.585C54.2801 56.8969 62.2555 24.5285 82.8587 9.84595C103.462 -4.83663 132.041 -0.498594 144.336 6.1753C154.172 11.5144 156.632 20.6354 156.632 24.5285V53.8937C158.958 57.3419 164.142 65.0391 166.269 68.2426C168.395 71.446 165.382 72.9143 163.61 73.248H156.632V102.613C156.632 114.626 145.001 114.96 142.01 114.96H129.714V131.978C152.311 138.652 155.524 148.107 154.305 152Z" fill="url(#paint0_linear_1_2)"/>
```
Comment (Brandon-G-Tripp) [inline static/assets/logo.svg:3]: Everything is looking great on this pass.

**Review by Brandon-G-Tripp — COMMENTED** [review]
All of the functionality looks good to me. I enjoy the soft backgroiund main tone. It seems to go well with the warm svg and everything else.

**Review by Brandon-G-Tripp — COMMENTED** [review]
Feels like a real site now. Getting closer to some actual functionality.

**Review by Brandon-G-Tripp — APPROVED** [review]

**Comment by brianfeister** [comment]
@j4inam I'm going to merge, if you need me to back this out, let me know and we can discuss reverting if yours needs to go first. 

🚀

**Pull request merged**